### PR TITLE
Feature/runner v2

### PR DESCRIPTION
**Pull request by oldani**

### CODE REVIEW

**Comment by wacevedo** [comment]
Hi Ordani, i see there are some extra unnecessary line on the template/report_template.html as you see here http://www.w3schools.com/html/html5_syntax.asp Blank Lines and Indentation section.

File: LICENCE.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+The MIT License (MIT)
+Copyright (c) 2016-2020 Ordanis Sanchez Suero
```
Comment (wacevedo) [inline LICENCE.txt]: This License  only require 1 date year, that is the  current one when is edited. I think you can delete the other one.

File: result.py [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+        tests = self.sort_test_list(tests)
+
+        for test in tests:
+            self._report_testcase(test, test_cases_list)
+
+        html_file = render_html('report_template.html', title=report_name,
+                                header=report_header, class_name=class_name,
+                                reportCases=test_cases_list, total_test=total_test)
+        return html_file
+
```
Comment (wacevedo) [inline result.py]: This is an necessary line?

**Comment by oldani** [comment]
Hi @wacevedo thanks men, already applied the feedbacks

**Pull request merged**